src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -325,7 +327,8 @@ protected async Task<(Guid, string projectName, SourceText)> GetGuidAndProjectNa
                 RoslynDefinitionBucket definitionBucket,
                 DocumentSpan documentSpan,
                 HighlightSpanKind spanKind,
-                ImmutableDictionary<string, ImmutableArray<string>> customColumnsDataOpt)
+                ImmutableDictionary<string, ImmutableArray<string>> customColumnsDataOpt,
```

Comment:
nit: rename `customColumnsDataOpt` to `propertiesWithMultipleValuesMapOpt`

Reply:
Also fine if you use the same terms as in the below method:
`private ImmutableDictionary<string, string> GetCustomColumnsData(IEnumerable<KeyValuePair<string, ImmutableArray<string>>> customColumnsWithMultipleValuesOpt, ImmutableArray<FindUsageProperty> customColumns)`